tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/DbScopeFactory.cs

```diff
@@ -66,9 +66,10 @@ private static Scope CreateDbCommandScope(Tracer tracer, IDbCommand command, Int
                     IastModule.OnSqlQuery(commandText, integrationId);
                 }
 
-                if (tracer.Settings.DbmPropagationMode != DbmPropagationLevel.Disabled && (integrationId == IntegrationId.MySql || integrationId == IntegrationId.Npgsql))
+                if (tracer.Settings.DbmPropagationMode != DbmPropagationLevel.Disabled && (integrationId == IntegrationId.MySql || integrationId == IntegrationId.Npgsql || integrationId == IntegrationId.SqlClient))
                 {
-                    command.CommandText = $"{DatabaseMonitoringPropagator.PropagateSpanData(tracer.Settings.DbmPropagationMode, tracer.DefaultServiceName, scope.Span.Context)} {commandText}";
+                    var propagationMode = integrationId != IntegrationId.SqlClient ? tracer.Settings.DbmPropagationMode : DbmPropagationLevel.Service;
```

Comment:
nit: flip the negatives is easier to read I think
```suggestion
                    var propagationMode = integrationId == IntegrationId.SqlClient ? DbmPropagationLevel.Service : tracer.Settings.DbmPropagationMode;
```